Formula/xsimd.rb

```diff
@@ -0,0 +1,42 @@
+class Xsimd < Formula
+  desc "Modern, portable C++ wrappers for SIMD intrinsics"
+  homepage "https://xsimd.readthedocs.io/en/latest/"
+  url "https://github.com/QuantStack/xsimd/archive/6.1.5.tar.gz"
+  sha256 "ecc781520f7b2bfdeb51d1385683694f2e89be1a05b77b222b9783cfeb645034"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args, "-DBUILD_TESTS=OFF"
```

Comment:
Would prefer:
```ruby
args = std_cmake_args
args << "-DBUILD_TESTS=OFF"

system "cmake", ".", *args
```
personally.

Reply:
I've patched the formula accordingly but used `cmake_args` instead of `args` just to be a little more verbose.

Reply:
Let's stick to `args` for consistency.

Reply:
I'm very sorry for not noticing that `args` was _de facto_. Modifications were made accordingly.

Reply:
No worries! I appreciate it's a small thing for me/us to be picky about.

Reply:
If my couple of seconds makes the life of everyone else easier I'm more than happy to comply =)